pysparkling/context.py

```diff
@@ -322,14 +322,6 @@ def union(self, rdds):
             (x for rdd in rdds for x in rdd.collect())
         )
 
-    def version(self):
```

Comment:
`Context()` already has a `version` attribute, making it impossible to reach the `version()` method. So using `version()` will always throw an exception, because it will resolve `version` first as the string attribute, and try to apply `__call__` on it, which will produce a `TypeError`:
```
>>> from pysparkling import Context
>>> Context().version
'0.3.16'
>>> Context().version()
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: 'str' object is not callable
``` 
Since the method is unreachable, better remove it.